webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -0,0 +1,260 @@
+import React from 'react';
+import {GitPullRequestIcon, IssueOpenedIcon, IconProps} from '@primer/octicons-react';
+import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
+import {Badge, Tooltip, OverlayTrigger} from "react-bootstrap";
+import * as CSS from 'csstype';
+import CrossIcon from "../../images/icons/cross";
```

Comment:
nit: it is a good practice to separate imports by assiciation groups. Also, it would be nice to use absolute paths.